app/customers_shops/customers.py

```diff
@@ -0,0 +1,65 @@
+from app.customers_shops.shops import Shop
+from dataclasses import dataclass
+import datetime
+
+
+@dataclass
+class Customer:
+    FUEL_PRICE: float

```

Comment:
it's not logical that the `Customer` class has a `fuel_price` attribute. Your class should be a model for real objects. Same with `shops_price`

Reply:
and why do you write class attributes with uppercase letters?